docs/governance/governance.md

```diff
@@ -23,149 +23,149 @@ The ODS Governance is designed with the following principles in mind:
+- TODS Project resource management, and
 - Internal and external project communications.
 
-- The [ODS Manager](#ods-manager) MAY be an individual or organization.
-- The [ODS Manager](#ods-manager) MUST assign an individual as the [ODS Program Manager](#ods-program-manager).
-- The [ODS Manager](#ods-manager) MUST communicate the name of the [ODS Program Manager](#ods-program-manager) to the [ODS Board](#ods-board-of-directors)in writing and notify the Board of any change in this assignment in writing.
-- The [ODS Manager](#ods-manager) SHOULD select a new [ODS Program Manager](#ods-program-manager) if asked to do so by the ODS Board.
+- The [TODS Manager](#tods-manager) MAY be an individual or organization.
+- The [TODS Manager](#tods-manager) MUST assign an individual as the [TODS Program Manager](#tods-program-manager).
+- The [TODS Manager](#tods-manager) MUST communicate the name of the [TODS Program Manager](#tods-program-manager) to the [TODS Board](#tods-board-of-directors) in writing and notify the Board of any change in this assignment in writing.
+- The [TODS Manager](#tods-manager) SHOULD select a new [TODS Program Manager](#tods-program-manager) if asked to do so by the TODS Board.
 
-## ODS Program Manager
+## TODS Program Manager
 
-Serves as the main contact between the [ODS Board](#ods-board-of-directors) and the [ODS Manager](#ods-manager).
+Serves as the main contact between the [TODS Board](#tods-board-of-directors) and the [TODS Manager](#tods-manager).
 
-## ODS Contributor
+## TODS Contributor
 
-Individuals apply to become Contributors by acknowledging and agreeing to the [ODS Contributor Agreement](#ods-contributor-agreement) and [ODS Code of Conduct](#ods-code-of-conduct).
+Individuals apply to become Contributors by acknowledging and agreeing to the [TODS Contributor Agreement](#tods-contributor-agreement) and [TODS Code of Conduct](#tods-code-of-conduct).
 
-- An ODS Contributor MUST abide by the [ODS Code of Conduct](#ods-code-of-conduct) or face removal.
-- An ODS Contributor MAY create issues, discussions, and pull requests in the [ODS Repository](#ods-repository).
-- An ODS Contributor MAY vote in decisions on changes to the ODS spec and other aspects of ODS.
+- An TODS Contributor MUST abide by the [TODS Code of Conduct](#tods-code-of-conduct) or face removal.
+- An TODS Contributor MAY create issues, discussions, and pull requests in the [TODS Repository](#tods-repository).
+- An TODS Contributor MAY vote in decisions on changes to the TODS spec and other aspects of TODS.
 
-## ODS Stakeholder
+## TODS Stakeholder
 
-Anyone interested in or could be directly affected by the ODS Project. Interested parties may register for the [ODS distribution list](https://groups.google.com/g/ods-stakeholders).
+Anyone interested in or could be directly affected by the TODS Project. Interested parties may register for the [TODS distribution list](https://groups.google.com/g/tods-stakeholders).
```

Comment:
This URL doesn't resolve (even before changing it from `ods-` to `tods-`. @safrazier17, what should it be? Is it https://groups.google.com/g/ods-working-group?

Reply:
Yes, it should

Reply:
Fixed in https://github.com/cal-itp/operational-data-standard/pull/78/commits/0da44bf7dc2821ec1521e0d07f0a8fb4c58f30d2